dask/array/core.py

```diff
@@ -663,8 +663,9 @@ def dtype(self):
 
     def __repr__(self):
         chunks = '(' + ', '.join(map(repr_long_list, self.chunks)) + ')'
-        return ("dask.array<%s, shape=%s, chunks=%s, dtype=%s>" %
-                (self.name, self.shape, chunks, self._dtype))
+        name = self.name if len(self.name) < 10 else self.name[:7] + '...'
+        return ("dask.array<%s, shape=%s, dtype=%s>" %
```

Comment:
I really like seeing some chunks info, because as a user it's often important to keep track of chunks with dask.array. As a compromise, maybe we could simply show the size of the first chunk?